workflows/files/ZWECONF.properties

```diff
@@ -539,6 +539,22 @@ components_caching_service_storage_vsam_name=
 #  Port for jgroups. This is required if storage mode is infinispan.
 components_caching_service_storage_infinispan_jgroups_port=7600
 
+# components_caching_service_storage_infinispan_jgroups_host
+# Label: Caching service storage jgroups host
+# Abstract: Host for jgroups
+# Category: components
+# Description:
+#  Host for jgroups
+components_caching_service_storage_infinispan_jgroups_host=
```

Comment:
Please add the default here

Reply:
there is no default value

Reply:
![Screenshot_20240904_143322_GitHub.jpg](https://github.com/user-attachments/assets/d65f9197-55f2-4b9d-b40b-009edecf8ae0)

What's this then?